src/DeviceRenderer.js

```diff
@@ -827,19 +830,47 @@ module.exports = class DeviceRenderer {
         }
     }
 
+    /**
+     * This function wraps around the plain js `addEventListener` function, also registering everything in a local array.
+     * The aim is to be able to remove all listeners at a later time.
+     * @param {EventTarget} object Object which emits the event
+     * @param {String} event A case-sensitive string reprensenting the event type to listen for
+     * @param {any} handler The object that receives a notification when an event of the specified type occus. This must be either `null`, an object with a `handleEvent()` method, or a function.
+     * @param {any} options Either a bool, specifying the `useCapture` arg, or an object specifying the `options` arg. Refer to the js api.
+     * @return {function} A removeListener function. This must be saved on the caller side if you ever want to use it to remove the listener
+     */
+    addListener(object, event, handler, options = {}) {
+        object.addEventListener(event, handler, options);
+        const id = new Date().getTime(); // TODO replace me with a proper uid once jérémy's fingerprint PR is merged
+        this.allListeners.push({id, object, event, handler, options});
+        return () => {
+            const index = this.allListeners.findIndex((item) => item.id === id);
+            if (index !== -1) {
```

Comment:
The doc says that the `findIndex` function only returns a valid index or -1, I'm not sure how `> -1` would be better? Is it because it's a faster comparison? Even so, I don't really think that the time gained would be significant.

Reply:
It's not for performance reasons, but rather to make it easily readable. I mean, 'a valid index' is a positive integer, so '> -1' seems clearer to me